packages/sdk/svelte/package.json

```
@@ -0,0 +1,89 @@
+{
+  "name": "@launchdarkly/svelte-client-sdk",
+  "version": "1.0.0",
```

```suggestion
  "version": "0.1.0",
```
We will want to do some prereleases to validate things. So we should drop this to 0.1.0.